docs/sources/flow/reference/components/prometheus.exporter.azure.md

```diff
@@ -119,8 +137,11 @@ prometheus.exporter.azure "example" {
 		"SuccessServerLatency",
 		"Transactions",
 	]
+	included_dimensions = [
+        "ApiName",
+        "TransactionType",
```

Comment:
nit: indent

Reply:
Oddly, locally this is all spaced properly and renders as I would expect. I'm really confused about why it's off when pushed.